app/javascript/src/components/Dashboard/Notes/Table.jsx

```diff
@@ -1,34 +1,36 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
 
-import { Table as NeetoUITable } from "neetoui";
-
-import { NOTES_TABLE_COLUMN_DATA } from "./constants";
+import Note from "./Note";
 import EditNotePane from "./Pane/Edit";
 
+import { DashboardContext } from "..";
+
 const Table = ({
-  selectedNoteIds,
-  setSelectedNoteIds,
+  // selectedNoteIds,
+  // setSelectedNoteIds,
```

Comment:
Go through the Slack thread. This is such a bad practice.

Reply:
I kept it as it would be needed in the `edit` functionality in the following tasks but I should've modified it with `TODO:` as you had said in the Slack thread. Sorry that I missed it.